apps/web/cypress/tests/digest-playground.spec.ts

```diff
@@ -4,12 +4,13 @@ describe('Digest Playground Workflow Page', function () {
   });
 
   it('should have a link to the docs', function () {
-    cy.intercept('**/notification-templates**').as('notificationTemplates');
-    cy.visit('/templates');
+    cy.intercept('GET', '**/notification-templates**').as('notificationTemplates');
```

Comment:
now there is only one path to the digest playground page - through the `/get-started`, so I've updated the tests to reflect that